apps/trustlab/src/payload/collections/Pages.js

```diff
@@ -1,23 +1,20 @@
 import { slug, fullTitle } from "@commons-ui/payload";
 
+import { canManagePages } from "@/trustlab/payload/access/abilities";
+import { anyone } from "@/trustlab/payload/access/anyone";
 import TestBlock from "@/trustlab/payload/blocks/TestBlock";
 
 const Pages = {
   slug: "pages",
   access: {
-    read: ({ req }) => {
-      if (req.user) return true;
-      return {
-        _status: {
-          equals: "published",
-        },
-      };
-    },
-    create: () => true,
-    update: () => true,
+    read: anyone,
+    create: ({ req: { user } }) => canManagePages(user),
+    update: ({ req: { user } }) => canManagePages(user),
+    delete: ({ req: { user } }) => canManagePages(user),
+    admin: ({ req: { user } }) => canManagePages(user),
```

Comment:
Can you add a comment why we might need this here? It's been a while since I read the docs but my old understanding was that `admin` access can **_only_** be used on collection that supports authentication. In our case that would be `Users` collection and not `Pages`.

Reply:
You are very right on this @kilemensi. It works only for auth collections